test/Microsoft.AspNetCore.Mvc.Core.Test/DependencyInjection/ApiBehaviorOptionsSetupTest.cs

```diff
@@ -5,12 +5,12 @@
 using System.Diagnostics;
 using System.Linq;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Infrastructure;
 using Microsoft.Extensions.Logging.Abstractions;
-using Microsoft.Extensions.Options;
```

Comment:
Why remove this `using`? Seems to make the `Options.Create(...)` calls messier.

Reply:
Since the type is is `Microsoft.Extension.DependencyInjection`, doing `Options.` inside refers to the namespace and not the type. Kinda have to namespace qualify `Options`